website/static/templates.json

```diff
@@ -1102,5 +1102,19 @@
           "bicep",
           "spring"
         ]
+    },
+    {
+      "title": "Bootstrap DevCenter with Azure Deployment Environments",
```

Comment:
@lopezleandro03 thanks for contribution.

- Seems more of a template(configuring/bootstrapping ADE env) for the platform admins?
- All features consumed in this template released/public?
- Just thinking if this fit into this repo as a developer template?

@savannahostrowski @wbreza please chime-in?

Reply:
Hi @rajeshkamal5050, 

Yes, it addresses the configuration of ADE for Platform Admins, depending on the organization topology it could be developers or a separate team. The Terraform code models projects after teams and creates different environment types for inspiration hooking up the Microsoft official ADE templates by default.

Azure Deployment Environments went GA on last Microsoft Build. All Terraform providers in use are stable.

For developers in DevOps teams managing their own infrastructure, this can give them a quick start with Azure Deployment Environments and further socialize this new service. Also, many users of Awesome AZD are platform engineers looking for infrastructure building blocks.

I am planning to extend the repository with:
- CI/CD pipelines examples to self-serve environments on-demand or based on triggers(PR creation for example) using GitHub Actions.
- Advanced ADE configuration options.
- In the future, expand the repo to cover the setup of DevCenter to enable Azure Dev Box.

I would love your feedback on how to make it more relevant for developers, happy to extend it.

Thanks,
Leandro